src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -24,13 +25,15 @@ public class PersonBuilder {
     public static final String DEFAULT_ROOMNUMBER = "sw-01-01 2023-10-20";
     public static final String DEFAULT_TELEGRAM = "amyBee";
     public static final String DEFAULT_BIRTHDAY = "01/01/2000";
+    public static final String DEFAULT_DORMTAG = "PGPR";
 
     private Name name;
     private Phone phone;
     private Email email;
     private RoomNumber roomNumber;
     private Telegram telegram;
     private Birthday birthday;
+    private DormTag dormTag;
```

Comment:
Might want to set the initialiser to initialise dormTag to null. Same for NewPersonBuilder.
Unless there was a reason for not initialising the field??